src/kubernetes_api_objects/secret.rs

```diff
@@ -0,0 +1,233 @@
+// Copyright 2022 VMware, Inc.
+// SPDX-License-Identifier: MIT
+use crate::kubernetes_api_objects::api_resource::*;
+use crate::kubernetes_api_objects::common::*;
+use crate::kubernetes_api_objects::dynamic::*;
+use crate::kubernetes_api_objects::error::ParseDynamicObjectError;
+use crate::kubernetes_api_objects::marshal::*;
+use crate::kubernetes_api_objects::object_meta::*;
+use crate::kubernetes_api_objects::resource::*;
+use crate::pervasive_ext::string_map::*;
+use crate::pervasive_ext::string_view::*;
+use deps_hack::k8s_openapi::ByteString;
```

Comment:
Nit: instead of importing `deps_hack::k8s_openapi::ByteString`, can you just use the full path of `ByteString` below?